src/ex1/script.js

```diff
@@ -0,0 +1,158 @@
+const todoListElement = document.getElementById("todo-list")
+const addTodoButton = document.getElementById("add-todo-button")
+const todoInput = document.getElementById("todo-input")
+const clearAllTodosButton = document.getElementById("clear-all-todos-button")
+const sumTodos = document.getElementById("sum-todos")
+const orderSelect = document.getElementById("order-select")
+const enterTodos = document.getElementById("enter-todos")
+
+let todoList = [] //array for storing data
+
+showTodos()
+
+todoInput.onkeyup = (e) => {
+    let enterValue = todoInput.value
+    if (enterValue.trim() !== ""){
+        addTodoButton.classList.add("active")
+        addTodoButton.style.cursor = "pointer"
+        addTodoButton.style.opacity = 1
+
+        if(e.keyCode === 13){
```

Comment:
try to avoid "magic numbers", creating a const for this value will improve readability, for example:

const ENTER_KEY = 13; // at the top of the file

if (e.keyCode === ENTER_KEY) {...}